customerio-reactnative-richpush.podspec

```diff
@@ -15,17 +15,23 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0" }
   s.source       = { :git => "https://github.com/customerio/customerio-ios.git", :tag => "#{s.version}" }
 
-  s.source_files = "ios/**/*.{h,m,mm,swift}"
+  s.source_files = "ios/wrappers/**/*.{h,m,mm,swift}"
 
   # s.dependency "X", "X"
   # Careful when declaring dependencies here. All dependencies will be included in the App Extension target in Xcode, not the host iOS app.   
 
   # Subspecs allow customers to choose between multiple options of what type of version of this rich push package they would like to install. 
   s.subspec 'apn' do |ss|
     ss.dependency "CustomerIO/MessagingPushAPN", package["cioNativeiOSSdkVersion"]
-  end
+    ss.resource_bundles = {
+      'CustomerIO_NSEResources' => ['ios/resources/*']
```

Comment:
Having resources with the same name was causing [failures in CI](https://github.com/customerio/customerio-reactnative/actions/runs/11252729899/job/31286497643#step:17:366), so I added resource with different name for NSE to avoid any risks.